apel_rest/urls.py

```diff
@@ -11,6 +11,6 @@
     # url(r'^blog/', include('blog.urls')),
 
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^accounting-server/api/v1/cloud/record$', CloudRecordView.as_view()),
-    url(r'^accounting-server/api/v1/cloud/record/summary$', CloudRecordSummaryView.as_view())
+    url(r'^api/v1/cloud/record$', CloudRecordView.as_view()),
```

Comment:
You're removing `accounting-server` while adding it to the `STATIC_URL` in `settings.py`?